frontend/src/components/Banner/Banner.tsx

```diff
@@ -45,7 +47,7 @@ export const Banner = ({
 
   return (
     <Collapse style={{ overflow: 'visible' }} in={isOpen} animateOpacity>
-      <Box __css={styles.banner}>
+      <Box __css={styles.banner} bgColor={bannerColor}>
```

Comment:
Might need to optionally add `bgColor` in the event that `__css={styles.banner}` has a default value. Not exactly sure how `__css` interacts with `bgColor: undefined`

Reply:
Thanks, done here [f14df7e](https://github.com/opengovsg/FormSG/pull/6431/commits/f14df7ed9f3f3dff84b57d81f7b36c67f75d330f)!